### PR TITLE
Updates & todos

### PR DESCRIPTION
**Pull request by ellieCrome**
- removes some unused files 
- updates a tonne of spelling mistakes
- adds new todos to the readme

### CODE REVIEW

File: exercises/advanced_async.test.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
-
 // ## Implement a node.js library wrapper for:
 
 // * Think about your SDK function signatures
-// * Think about behaviours you want to expose from your SDK
+// * Think about behaviors you want to expose from your SDK
```
Comment (andydavison) [inline exercises/advanced_async.test.js]: 🇺🇸

File: src/objects_and_arrays.js [hunk 2]
```diff
@@ -6,23 +6,24 @@
...
 new Array();
 new Object();
 
 // and their counterpart:
 
-[]
-{}
+[];
+{
+}
```
Comment (andydavison) [inline src/objects_and_arrays.js]: does this need to change?

File: src/promises.js [hunk 3]
```diff
@@ -82,45 +84,41 @@ gitHubData
...
 
 // ## Promise return values
 
 // Promises work heavily with return values, this is a *very common gotcha*
 
-// A consuming function _must_ have access to the Promise in order to understand when it is returned. 
+// A consuming function _must_ have access to the Promise in order to understand when it is returned.
 
-// This is common to fail to recognise this when a function expects a promise, such as in a jest test. 
+// This is common to fail to recognize this when a function expects a promise, such as in a jest test.
```
Comment (andydavison) [inline src/promises.js]: 🇺🇸

File: src/objects_and_arrays.js (hunk 2 above)
Comment (ellieCrome) [reply src/objects_and_arrays.js]: no !

**Pull request merged**